File: SDKs/amazonaws.com/elasticache/2015-02-02/python/src/sdk/models/shared/updateactionresultsmessage.py
import dataclasses
from datetime import date, datetime
from marshmallow import fields
import dateutil.parser
from typing import Optional
from ..shared import processedupdateactionlist as shared_processedupdateactionlist
from ..shared import unprocessedupdateactionlist as shared_unprocessedupdateactionlist


@dataclasses.dataclass
class UpdateActionResultsMessage:
    processed_update_actions: Optional[list[shared_processedupdateactionlist.ProcessedUpdateActionList]] = dataclasses.field(default=None)
    unprocessed_update_actions: Optional[list[shared_unprocessedupdateactionlist.UnprocessedUpdateActionList]] = dataclasses.field(default=None)
    

File: SDKs/amazonaws.com/elasticache/2015-02-02/typescript/src/sdk/models/shared/updateactionresultsmessage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { ProcessedUpdateActionList } from "./processedupdateactionlist";
import { UnprocessedUpdateActionList } from "./unprocessedupdateactionlist";



export class UpdateActionResultsMessage extends SpeakeasyBase {
  @SpeakeasyMetadata({ elemType: ProcessedUpdateActionList })
  processedUpdateActions?: ProcessedUpdateActionList[];

  @SpeakeasyMetadata({ elemType: UnprocessedUpdateActionList })
  unprocessedUpdateActions?: UnprocessedUpdateActionList[];
}
